.npmignore

```diff
@@ -1,7 +1,8 @@
-*
+**/*
 !LICENSE
 !README.md
 !bin/*
 !lib/**/*
+!src/**/*
```

Comment:
do we need to include src? shouldn't to .js and .d.ts be enough?

Reply:
personally I find the src/ useful as a consumer

Reply:
I think we were providing the src before (as we were providing literally all subdirectories), if there isn't too much of a downside then preserving this behaviour seems OK?